zp-relayer/router.ts

```diff
@@ -25,7 +25,11 @@ function wrapErr(f: (_req: Request, _res: Response, _next: NextFunction) => Prom
 export function createRouter({ feeManager }: IRouterConfig) {
   const router = express.Router()
 
-  router.use(cors())
+  router.use(
+    cors({
+      origin: config.corsOrigin,
+    })
```

Comment:
Do you think it makes sense to only limit origin in requests for sending transactions?
So that any read-only integrations won't break

Reply:
Hmm, didn't think about possible read-only integrations.
@akolotov wdyt?